site/content/reference/01-commands.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Commands Reference"
+weight: 10
+---
+
+## Commands Reference
+
+```
+a CLI for Amazon EKS
+
+Usage: eksctl [command] [flags]
+
+Commands:
+  eksctl completion  Generates shell completion scripts
+  eksctl create      Create resource(s)
+  eksctl delete      Delete resource(s)
+  eksctl drain       drain resources(s)
+  eksctl get         Get resource(s)
+  eksctl help        Help about any command
+  eksctl scale       Scale resources(s)
+  eksctl update      Update resource(s)
+  eksctl utils       Various utils
+  eksctl version     Output the version of eksctl
+
+Use 'eksctl [command] --help' for more information about a command.
+```
+
+`Usage: eksctl [command] [flags]`
+
+##### common flags<a name="common-flags"></a>
+
+These flags can be applied to all eksctl commands and behave consistently between them.
+
+- [C, color](02-flags.md#color)
+- [h, help](02-flags.md#help)
+- [v, verbose](02-flags.md#verbose)
+
+### completion <a name="completion"></a>
+
+Generates shell completion scripts for various Unix command-line shells.
+
+`Usage: eksctl completion [command]`
+
+The `completion` commands do not support any additional flags.
+
+Without an invoking a sub-command, the completion command simply prints the help screen.
+
+#### bash <a name="completion-bash"></a>
```

Comment:
Same as before.

Reply:
`bash` users are people too!!!  😄 

`zsh` is an even more obscure shell, but someone liked it enough to write completion support for it, so who am I to judge?